Libraries/LibRegex/RegexParser.cpp

```diff
@@ -967,7 +967,7 @@ bool ECMA262Parser::parse_atom(ByteCode& stack, size_t& match_length_minimum, bo
         return false;
     }
 
-    if (match(TokenType::Char)) {
+    if (match_ordinary_characters()) {
```

Comment:
What group of tokens aren't allowed here? I know a bunch of them are excluded above, and a few more are already checked above.
(this change is alright, I'm just wondering out loud)